src/renderer/hooks/useCurrency.ts

```
@@ -0,0 +1,132 @@
+      setRates(currentRates);
+    } catch (error) {
+      console.error('Failed to load currency rates:', error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const convertFromUSD = useCallback(
+    (usdAmount: number): number => {
+      // Validate input amount
+      if (typeof usdAmount !== 'number' || !isFinite(usdAmount)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)